src/test/java/org/broadinstitute/hellbender/tools/walkers/vqsr/VariantRecalibratorIntegrationTest.java

```diff
@@ -187,6 +257,11 @@ public void testVariantRecalibratorSNP(final String[] params, final String tranc
         doSNPTest(params, tranchesPath, recalPath);
     }
 
+    @Test(dataProvider = "BadVarRecal", expectedExceptions = {UserException.BadInput.class})
```

Comment:
See comment above.

Reply:
The "bad" cases are covered in other more specific tests.  I'll take it out.